source/MaterialXTest/MaterialXGenShader/GenShaderUtil.cpp

```diff
@@ -996,6 +996,7 @@ bool TestSuiteOptions::readOptions(const std::string& optionFile)
     MaterialX::DocumentPtr doc = MaterialX::createDocument();
     try {
         mx::XmlReadOptions readOptions;
+        readOptions.applyFutureUpdates = true;
```

Comment:
Should this not be set by default in the constructor of XmlReadOptions instead?

Reply:
Yes, it would make it consistent:
1) If no options then apply. by default
2) If options then apply by default
I will probably leave these in to make it explicit in case we flip logic again, this should always be set to true.